spec/voodoo_sms_spec.rb

```diff
@@ -54,16 +54,20 @@
 
     context '200 success', vcr: :success do
       let(:vcr_cassette) { 'send_sms' }
-      it { expect(client.send_sms(orig, dest, msg)).to eq true }
+      it { expect(client.send_sms(orig, dest, msg).reference_id).to eq("4103395") }
     end
 
     context '200 success - multipart message', vcr: :success do
       let(:vcr_cassette) { 'send_multipart_sms' }
-      it { expect(client.send_sms(orig, dest, 'A'*320)).to eq true }
+      it { expect(client.send_sms(orig, dest, 'A'*320).reference_id).to eq("4103395") }
     end
 
     context 'validation' do
-      before(:each) { allow(VoodooSMS).to receive(:get).and_return({'result' => 200}) }
+      before(:each) do
+        response = double("HTTParty::Response", parsed_response: {})
+        allow(response).to receive(:[]).with("result").and_return("200 OK")
```

Comment:
These test cases test validations that happen before submitting the request to Voodoo, so it does not matter what Voodoo responds. But maybe it's better to return valid response from the mocked method too - I will update it.